src/ImportPage.js

```diff
@@ -76,6 +80,25 @@ export default function ImportPage({onDataImported}: Props) {
                     performance profile
                   </a>{' '}
                   from Chrome Devtools.
+                  <br />
+                  Use <kbd>Ctrl + Scroll</kbd> or <kbd>Shift + Scroll</kbd> to
```

Comment:
This looks like they need to press the Scroll button on their keyboard (which I think may actually exist?). Maybe we should replace this with something like "To zoom, scroll while holding down `Ctrl` or `Shift`."

Reply:
Ah, that's a good point.